includes/rendering/MWPDFRenderingAPI.php

```diff
@@ -7,23 +7,33 @@
  */
 class MWPDFRenderingAPI extends CollectionRenderingAPI {
 	protected function makeRequest( $command, array $params ) {
-        $params['command'] = $command;
-        $serveURL = "http://localhost:3333/";
+		global $wgSimpleBookPassthroughParameters,
+			$wgSimpleBookRenderingApiUrl,
+			$wgSimpleBookRenderingCredentials;
 
-		if ( !$serveURL ) {
-			wfDebugLog( 'collection', 'The mwlib/OCG render server URL isn\'t configured.' );
+		$params['command'] = $command;
 
+		if ( !$wgSimpleBookRenderingApiUrl ) {
+			wfDebugLog( 'collection', 'The SimpleBook rendering API URL isn\'t configured.' );
 			return new CollectionAPIResult( null );
 		}
 
+		if ( $wgSimpleBookRenderingCredentials ) {
+			$params['login_credentials'] = $wgSimpleBookRenderingCredentials;
+		}
+
+		if ( $wgSimpleBookRenderingCredentials ) {
```

Comment:
This looks suspicious? Should it be `wgSimpleBookPassthroughParameters`?

Reply:
excellent catch thank you!